src/UniformSession/UniformSessionFeature.cs

```diff
@@ -10,7 +11,7 @@ protected override void Setup(FeatureConfigurationContext context)
         context.Pipeline.Register(new RegisterCurrentSessionBehavior(sessionHolder), "Enables floating of uniform session.");
         context.RegisterStartupTask(new RegisterSessionStartupTask(sessionHolder));
 
-        context.Container.ConfigureComponent(() => sessionHolder.Current, DependencyLifecycle.InstancePerCall);
+        context.Services.AddTransient((s) => sessionHolder.Current);
```

Comment:
```suggestion
        context.Services.AddTransient(_ => sessionHolder.Current);
```

that also seems to cause the inspection error